src/arena.rs

```diff
@@ -58,6 +58,19 @@ pub struct ArenaFull {
     standing: Vec<JsValue>,
 }
 
+impl ArenaFull {
+    fn user_from_id(&self, uid: &UserId) -> ClientMe {
+        ClientMe::new(
+            self.ranking.ranking.get(&uid),
+            self.ongoing_user_games.get(&uid)
+        )
+    }
```

Comment:
I don't think we can do that because eventually `ClientMe` will need more data than what `ArenaFull` has.